repository/kafka/operator/params.yaml

```diff
@@ -104,14 +104,34 @@ LEADER_IMBALANCE_PER_BROKER_PERCENTAGE:
 
 LIVENESS_INITIAL_DELAY_SECONDS:
   displayName: "livenessProbe initial delay timeout seconds"
-  description: "livenessProbe initial delay in seconds"
+  description: "Number of seconds after the container has started before liveness probes is initiated."
   default: "30"
 
 LIVENESS_PERIOD_SECONDS:
   displayName: "livenessProbe period seconds"
-  description: "livenessProbe period in seconds"
+  description: "How often (in seconds) to perform the probe. Default to 30 seconds. Minimum value is 1."
   default: "30"
 
+LIVENESS_TIMEOUT_SECONDS:
+  displayName: "livenessProbe timeout seconds"
+  description: "Number of seconds after which the probe times out. Defaults to 30 seconds. Minimum value is 1."
+  default: "30"
+
+LIVENESS_FAILURE_THRESHOLD:
+  displayName: "livenessProbe timeout seconds"
+  description: "When the Pod starts and the probe fails, Kubernetes will try failureThreshold times before restarting the Pod. Defaults to 3. Minimum value is 1."
+  default: "3"
+
+LIVENESS_METHOD:
+  displayName: "livenessProbe method"
+  description: "livenessProbe method values 'PORT' or 'FUNCTIONAL'"
+  default: "PORT"
+
+LIVENESS_TOPIC_PREFIX:
+  displayName: "livenessProbe topic prefix"
+  description: "This topic is used by livenessProbe when 'FUNCTIONAL' method is selected."
+  default: "LivenessProbeTopicPrefix"
```

Comment:
can we change the default to something without the word `Prefix` like `KafkaLivenessTopic` ?